tests/wpt/tests/html/semantics/embedded-content/the-iframe-element/multiple-iframes-with-allow-scripts-crash.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+    <body>
```

Comment:
Just to confirm my understanding, the styles in the original crash test didn't contribute to the cause of the crash, so now they are removed? i.e we are not simply deleting this test because this new test without the CSS is still valuable as a crash test?

Reply:
I think it's very hard to say now if the styles contributed to the crash, since this test was added so long ago. I am happy to re-add them though.